Bottom: 04370a3f04f3ddbfb1bebbc6a7b54b96615d6976
Top:    23ca2576e16cd4a56dcaa6a50550a95eebc7824c
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-17 16:12:01 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/roles/bacula-restores/README.md b/roles/bacula-restores/README.md
new file mode 100644
index 0000000..225dd44
--- /dev/null
+++ b/roles/bacula-restores/README.md
@@ -0,0 +1,38 @@
+Role Name
+=========
+
+A brief description of the role goes here.
+
+Requirements
+------------
+
+Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
+
+Role Variables
+--------------
+
+A description of the settable variables for this role should go here, including any variables that are in defaults/main.yml, vars/main.yml, and any variables that can/should be set via parameters to the role. Any variables that are read from other roles and/or the global scope (ie. hostvars, group vars, etc.) should be mentioned here as well.
+
+Dependencies
+------------
+
+A list of other roles hosted on Galaxy should go here, plus any details in regards to parameters that may need to be set for other roles, or variables that are used from other roles.
+
+Example Playbook
+----------------
+
+Including an example of how to use your role (for instance, with variables passed in as parameters) is always nice for users too:
+
+    - hosts: servers
+      roles:
+         - { role: username.rolename, x: 42 }
+
+License
+-------
+
+BSD
+
+Author Information
+------------------
+
+An optional section for the role authors to include contact information, or a website (HTML is not allowed).
diff --git a/roles/bacula-restores/defaults/main.yml b/roles/bacula-restores/defaults/main.yml
new file mode 100644
index 0000000..540bc12
--- /dev/null
+++ b/roles/bacula-restores/defaults/main.yml
@@ -0,0 +1,2 @@
+---
+# defaults file for bacula-restores
\ No newline at end of file
diff --git a/roles/bacula-restores/handlers/main.yml b/roles/bacula-restores/handlers/main.yml
new file mode 100644
index 0000000..66e6f98
--- /dev/null
+++ b/roles/bacula-restores/handlers/main.yml
@@ -0,0 +1,2 @@
+---
+# handlers file for bacula-restores
\ No newline at end of file
diff --git a/roles/bacula-restores/meta/main.yml b/roles/bacula-restores/meta/main.yml
new file mode 100644
index 0000000..7223799
--- /dev/null
+++ b/roles/bacula-restores/meta/main.yml
@@ -0,0 +1,57 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
+
+  min_ansible_version: 1.2
+
+  # If this a Container Enabled role, provide the minimum Ansible Container version.
+  # min_ansible_container_version:
+
+  # Optionally specify the branch Galaxy will use when accessing the GitHub
+  # repo for this role. During role install, if no tags are available,
+  # Galaxy will use this branch. During import Galaxy will access files on
+  # this branch. If Travis integration is configured, only notifications for this
+  # branch will be accepted. Otherwise, in all cases, the repo's default branch
+  # (usually master) will be used.
+  #github_branch:
+
+  #
+  # platforms is a list of platforms, and each platform has a name and a list of versions.
+  #
+  # platforms:
+  # - name: Fedora
+  #   versions:
+  #   - all
+  #   - 25
+  # - name: SomePlatform
+  #   versions:
+  #   - all
+  #   - 1.0
+  #   - 7
+  #   - 99.99
+
+  galaxy_tags: []
+    # List tags for your role here, one per line. A tag is a keyword that describes
+    # and categorizes the role. Users find roles by searching for tags. Be sure to
+    # remove the '[]' above, if you add tags to this list.
+    #
+    # NOTE: A tag is limited to a single word comprised of alphanumeric characters.
+    #       Maximum 20 tags per role.
+
+dependencies: []
+  # List your role dependencies here, one per line. Be sure to remove the '[]' above,
+  # if you add dependencies to this list.
\ No newline at end of file
diff --git a/roles/bacula-restores/tasks/main.yml b/roles/bacula-restores/tasks/main.yml
new file mode 100644
index 0000000..560f5ba
--- /dev/null
+++ b/roles/bacula-restores/tasks/main.yml
@@ -0,0 +1,26 @@
+---
+# tasks file for bacula-restores
+
+- name: List bacula-restores
+  find:
+    paths: /mnt/bacula-restores
+    file_type: file
+    recurse: yes
+  register: bacula_restores_list
+
+- name: Create bacula-restores symlinks
+  file:
+    path: '{{ item.path | replace("/mnt/bacula-restores","/tmp/bacula-restores") | dirname }}'
+    state: directory
+    force: yes
+    mode: 0777
+  with_items: '{{ bacula_restores_list.files }}'
+
+- name: Create bacula-restores directories
+  file:
+    src: '{{ item.path }}'
+    dest: '{{ item.path | replace("/mnt/bacula-restores","/tmp/bacula-restores") }}'
+    state: link
+    force: yes
+    mode: '{{ item.mode }}'
+  with_items: '{{ bacula_restores_list.files }}'
diff --git a/roles/bacula-restores/tests/inventory b/roles/bacula-restores/tests/inventory
new file mode 100644
index 0000000..878877b
--- /dev/null
+++ b/roles/bacula-restores/tests/inventory
@@ -0,0 +1,2 @@
+localhost
+
diff --git a/roles/bacula-restores/tests/test.yml b/roles/bacula-restores/tests/test.yml
new file mode 100644
index 0000000..3b1d784
--- /dev/null
+++ b/roles/bacula-restores/tests/test.yml
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - bacula-restores
\ No newline at end of file
diff --git a/roles/bacula-restores/vars/main.yml b/roles/bacula-restores/vars/main.yml
new file mode 100644
index 0000000..393c567
--- /dev/null
+++ b/roles/bacula-restores/vars/main.yml
@@ -0,0 +1,2 @@
+---
+# vars file for bacula-restores
\ No newline at end of file
